.github/workflows/ci_linux_arm_mu4.yml

```diff
@@ -3,7 +3,7 @@ name: CI_Linux_ARM MU4
 on:
   schedule:
     - cron: '0 3 */1 */1 *' # At 03:00 on every day-of-month for master
-    - cron: '0 3 */1 */1 *' # At 05:00 on every day-of-month for current release branch
+   # - cron: '0 3 */1 */1 *' # At 05:00 on every day-of-month for current release branch
```

Comment:
Is this intentionally disabled?

Reply:
Yes, I intentionally disabled it, as I wasn't sure if we needed to build it every day (given how resource-intensive this workflow is)... We can re-enable it if the need arises